integrationtest/neo/client/request/internal/GetAll.d

```diff
@@ -273,7 +274,7 @@ private scope class GetAllImpl
 
     private bool handleStatusCode ( ubyte status )
     {
-        auto request_status = cast(RequestStatusCode)status;
+        auto request_status = cast(SupportedStatus)status;
```

Comment:
This looks unrelated to this PR?

Reply:
Hm, I wanted to get rid of all the deprecations, but I should actually get rid of them in a good way - actually using the new mechanism.